conanfile.py

```diff
@@ -188,8 +201,14 @@ def configure(self):
         if self.settings.os != "Linux":
             self.options.with_libalsa = False
 
+        if (not self.settings.shared) and self.options["qtwebengine"]:
```

Comment:
`if (not self.options.shared) and self.options.qtwebengine:` ?

Reply:
Lol my bad